From: Andrew Morton <akpm@linux-foundation.org>
Subject: mm-page_owner-add-page_owner_stacks-file-to-print-out-only-stacks-and-their-counte-fix
Date: Wed May 17 12:58:14 PM PDT 2023

build fix

include/linux/stackdepot.h:116:26: error: declaration of 'struct seq_file' will not be visible outside of this function

Reported-by: kernel test robot <lkp@intel.com>
Closes: https://lore.kernel.org/oe-kbuild-all/202305171411.iw7APiY8-lkp@intel.com/
Cc: Oscar Salvador <osalvador@suse.de>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 include/linux/stackdepot.h |    1 +
 1 file changed, 1 insertion(+)

--- a/include/linux/stackdepot.h~mm-page_owner-add-page_owner_stacks-file-to-print-out-only-stacks-and-their-counte-fix
+++ a/include/linux/stackdepot.h
@@ -25,6 +25,7 @@
 #include <linux/gfp.h>
 
 typedef u32 depot_stack_handle_t;
+struct seq_file;
 
 /*
  * Number of bits in the handle that stack depot doesn't use. Users may store
_
